### ISSUE TITLE
Crash when trying to create a user

### ISSUE THREAD

**Issue by dhasenan** [shawncplus/ranviermud#222]
### Description
Ranvier crashes when I create a new user, connecting via the Linux `telnet` client.

### Expected outcome
A new user would be successfully created.

### Actual outcome
Telnet client:

```
Welcome, what is your name? dhasenan
Do you want your account's username to be Dhasenan? [y/n] y
Connection closed by foreign host.
```

Server log:

```
2017-06-28T03:29:18.862Z - info: Setting autosave to 10 seconds.
2017-06-28T03:29:18.862Z - info: Server started on port: 4000...
2017-06-28T03:29:18.967Z - verbose: 	SPAWN: Adding npc [limbo:4] to room [Training Room]
2017-06-28T03:29:25.791Z - info: User connected...
2017-06-28T03:29:29.100Z - error: No account found as Dhasenan.
buffer.js:557
  throw new TypeError('Unknown encoding: ' + encoding);
  ^

TypeError: Unknown encoding: 
    at stringSlice (buffer.js:557:9)
    at Buffer.toString (buffer.js:593:10)
    at TelnetStream.socket.once.data (/home/dhasenan/src/ranviermud/bundles/ranvier-input-events/input-events/create-account.js:19:21)
    at Object.onceWrapper (events.js:316:30)
    at emitOne (events.js:115:13)
    at TelnetStream.emit (events.js:210:7)
    at TelnetStream.input (/home/dhasenan/src/ranviermud/src/Telnet.js:244:10)
    at Socket.connection.on (/home/dhasenan/src/ranviermud/src/Telnet.js:163:16)
    at emitOne (events.js:115:13)
    at Socket.emit (events.js:210:7)
```

### Steps to reproduce
1. Install ranvier
2. Start ranvier
3. telnet to ranvier instance
4. Create a new user

### Version of Ranvier
version: 1.0.1, git ref: 72cf1721254c2d6aeee9364c019ac15f8163fabb

node: v8.1.2 OS: Ubuntu 16.04

**Comment by shawncplus** [comment]
Interesting, I haven't tried Ranvier with node 8 yet. Might have to update the README specifying node 7.x

**Comment by shawncplus** [comment]
My suggestion for now would be to use Node.js version 7.x until I can take a whack at a Node 8 compatibility pass.

**Comment by shawncplus** [comment]
Fixed in 5b36dab24f4db58de4b736e33f601ab85e9c5b6f
